proto/stride/auction/auction.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package stride.auction;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v24/x/auction/types";
+
+enum AuctionType {
+  // Default value - should not be used
+  AUCTION_TYPE_UNSPECIFIED = 0;
+  // First-Come First-Served auction
+  AUCTION_TYPE_FCFS = 1;
+}
+message Params {}
+
+message Auction {
+  // Auction type
+  AuctionType type = 1;
+
+  // A unique auction name
+  string name = 2;
+
+  // Token denom being sold in the auction
+  string selling_denom = 3;
+
+  // Token denom used to place bids
+  string payment_denom = 4;
+
+  // Whether auction is active
+  bool enabled = 5;
+
+  // Price multiplier (e.g. 0.95 for 5% discount)
+  string price_multiplier = 6 [
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
Sam:
i think we had a thread in the gdoc about this. "multiplier" is underspecified and kinda confusing imo - I think it should indicate that it's a lower bound

my preference would be either MinPriceMultiplier or FloorPriceMultiplier

fwiw, when we move onto different auction types, this field will still be relevant as the floor (and the term "discount" would make less sense)


Assaf:
I don't like "min" and "floor". They both sound like this multiplier number can be this or higher, which in my view is confusing.

I also agree that priceMultiplier is not that good, but at least it specifies what this number is used for.

claude suggests price_adjustment_factor, wdyt?

Reply:
> They both sound like this multiplier number can be this or higher, which in my view is confusing.

I think the important point though is that the price from the bids _can_ be higher than this value.

I just asked claude too and he spit out the following (I never used the word floor in my prompt)

```
minimumBidRatio
minPriceMultiplier
floorPriceRatio
bidPriceThreshold
priceDiscountFactor
minimumPricePercentage
reservePriceRatio
bidFloorMultiplier
priceFloorFactor
minimumBidPercentage

Some related terms commonly used in auction terminology:

"Floor price" is a common auction term for minimum acceptable price
```

my vote is `minPriceMultiplier` so you can keep your `multiplier` and i keep my `min` lol 😄

Reply:
never thought about it this way and now I fully agree that this is actually providing the floor price for bids

Reply:
59a9e7852